lib/bridge/IrcHandler.js

```diff
@@ -443,7 +443,10 @@ IrcHandler.prototype.onMetadata = Promise.coroutine(function*(req, client, msg)
         let response = yield this.ircBridge.getAppServiceBridge().getIntent().createRoom({
             createAsClient: false,
             options: {
-                name: "IRC Admin Room",
+                name: `${client.server.domain} Admin`,
```

Comment:
"Admin" sounds like you are the administrator for the network... Or at least in a room that connects you to an admin user.

I had other suggestions in #198